testgrid/web/.babelrc

```diff
@@ -11,6 +11,5 @@
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
     "@babel/plugin-proposal-object-rest-spread",
     "@babel/plugin-proposal-class-properties",
-    "react-hot-loader/babel",
```

Comment:
I removed this because it throws a warning in the browser, and I'm not sure it's working anyway. It's currently deprecated, but I couldn't get the replacement to work in a reasonable amount of time.